src/Components/Navbar.jsx

```diff
@@ -17,27 +17,31 @@ const Navbar = () => {
 
 
   return (
-    <div>
-      <Flex m={4} shadow={'sm'} p={5}>
-        <Box>
+    <DIV >
+      <Flex m={4} shadow={'sm'} p={5} >
+        <Box className='img'>
           {/* <img src="./image.dribbble-mania.jpeg"></img> */}
           <Link to='/'><Image w={110}  src='./dribbble-mania.jpg'/></Link>
         </Box>
-        <Box ml={5}>
-          <Link to="/products" style={{textDecoration: "none"}}><Text fontWeight={500} color={'grey'} _hover={{color:'black'}} as={'h2'}>Inspiration</Text></Link>
-        </Box>
-        <Box ml={5}>
-          <Link to="/products" style={{textDecoration: "none"}}><Text fontWeight={500} color={'grey'} _hover={{color:'black'}} as={'h2'}>Find Work</Text></Link>
-        </Box >
-        <Box ml={5}>
-          <Link to="/products" style={{textDecoration: "none"}}><Text fontWeight={500} color={'grey'} _hover={{color:'black'}} as={'h2'}>Learn Design</Text></Link>
-        </Box>
-        <Box ml={5}>
-          <Link to="/products" style={{textDecoration: "none"}}><Text fontWeight={500} color={'grey'} _hover={{color:'black'}} as={'h2'}>Go Pro</Text></Link>
+        <Box className='first'>
+          <Flex>
+            <Box ml={5}>
+              <Link to="/products" style={{textDecoration: "none"}}><Text fontWeight={500} color={'grey'} _hover={{color:'black'}} as={'h2'}>Inspiration</Text></Link>
+            </Box>
+            <Box ml={5}>
+              <Link to="/products" style={{textDecoration: "none"}}><Text fontWeight={500} color={'grey'} _hover={{color:'black'}} as={'h2'}>Find Work</Text></Link>
+            </Box >
+            <Box ml={5}>
+              <Link to="/products" style={{textDecoration: "none"}}><Text fontWeight={500} color={'grey'} _hover={{color:'black'}} as={'h2'}>Learn Design</Text></Link>
+            </Box>
+            <Box ml={5}>
+              <Link to="/products" style={{textDecoration: "none"}}><Text fontWeight={500} color={'grey'} _hover={{color:'black'}} as={'h2'}>Go Pro</Text></Link>
+            </Box>
+            <Box ml={5}>
+              <Link to="/products" style={{textDecoration: "none"}}><Text fontWeight={500} color={'grey'} _hover={{color:'black'}} as={'h2'}>Hire Designers</Text></Link>
```

Comment:
In the <Box> The majority of the things are same. For this case it's best to create a config and iterate over the config to render the same things. By that the code looks more readable.

Reply:
Also for the css it's better to add all the styles in the CSS file